test-requirements.txt

```diff
@@ -1,8 +1,10 @@
 ddt==0.8.0
 django-nose==1.4.1
+freezegun==0.1.11
 mock==1.0.1
 nose==1.3.3
 coverage==4.0.2
+python-dateutil==2.1
```

Comment:
When I just added `freezegun`, I got this weird error from pip:
```
Collecting python-dateutil<2.0,>=1.0,>=2.1 (from freezegun==0.1.11->-r test-requirements.txt (line 3))
  Could not find a version that satisfies the requirement python-dateutil<2.0,>=1.0,>=2.1 (from freezegun==0.1.11->-r test-requirements.txt (line 3)) (from versions: 1.4, 1.4.1, 1.5, 2.1, 2.2, 2.3, 2.4.0, 2.4.1, 2.4.1.post1, 2.4.2, 2.5.0, 2.5.1, 2.5.2, 2.5.3, 2.6.0)
No matching distribution found for python-dateutil<2.0,>=1.0,>=2.1 (from freezegun==0.1.11->-r test-requirements.txt (line 3))
```
Since no version will ever match `python-dateutil<2.0,>=1.0,>=2.1`, I [copied the version listed in edx-platform](https://github.com/edx/edx-platform/blob/master/requirements/edx/base.txt#L86) and that fixed things.